### PR TITLE
Expose firewall services to tenant network

### PR DESCRIPTION
**Pull request by mwindower**

### CODE REVIEW

File: pkg/nftables/nftables.tpl [hunk 1]
```diff
@@ -28,11 +20,11 @@ table ip firewall {
 		type filter hook forward priority 1; policy drop;
 
 		# network traffic accounting for external traffic
-		ip saddr != @internal_prefixes ip saddr != @cluster_prefixes counter name external_in
-		ip daddr != @internal_prefixes ip daddr != @cluster_prefixes counter name external_out
+		ip saddr != @internal_prefixes counter name external_in
+		ip daddr != @internal_prefixes counter name external_out
 
 		# network traffic accounting for internal traffic
-		ip daddr == @internal_prefixes ip saddr == @internal_prefixes counter name internal_total
```
Comment (majst01) [inline pkg/nftables/nftables.tpl:35]: missing `@`

File: pkg/nftables/nftables.tpl [hunk 2]
```diff
@@ -8,14 +8,6 @@ table ip firewall {
 		elements = { {{ .InternalPrefixes }} }
 		{{ end }}
 	}
-	# Prefixes in the cluster, typically 10.x.x.x
-	# FIXME Should be filled with nodeCidr
-	set cluster_prefixes {
```
Comment (majst01) [inline pkg/nftables/nftables.tpl:13]: can be deleted, still used: 
https://github.com/metal-stack/firewall-controller/blob/master/pkg/nftables/network_policy.go#L74

File: main.go [hunk 3]
```diff
@@ -57,11 +57,13 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var hostsFile string
+	var serviceIP string
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&hostsFile, "hosts-file", "/etc/hosts", "The hosts file to manipulate for the droptailer.")
+	flag.StringVar(&serviceIP, "service-ip", "172.17.0.1", "The ip where firewall services are exposed.")
```
Comment (majst01) [inline main.go:66]: couldn't we detect this from the firewall-controller itself ? With this change we have to adopt metal-networker as well

Comment (mwindower) [reply main.go:66]: how? i would not want to parse yaml files from `/etc/metal`

Comment (majst01) [reply main.go:66]: ok, you are right

**Pull request merged**